tests/integration/components/denali-multi-select-test.js

```diff
@@ -0,0 +1,161 @@
+import { module, test } from 'qunit';
+import { setupRenderingTest } from 'ember-qunit';
+import { render, setupOnerror, findAll, click, fillIn, triggerKeyEvent } from '@ember/test-helpers';
+import { hbs } from 'ember-cli-htmlbars';
+
+module('Integration | Component | denali-multi-select', function (hooks) {
+  setupRenderingTest(hooks);
+
+  test('it requires options', async function (assert) {
+    assert.expect(1);
+
+    setupOnerror(function ({ message }) {
+      assert.equal(
+        message,
+        'Failed prop type: The prop `options` is marked as required in `DenaliMultiSelectComponent`, but its value is `undefined`.',
+        'If @args() contains a PropType validator, an error will be thrown if the value is incorrect'
+      );
+    });
+    await render(hbs`<DenaliMultiSelect />`);
+  });
+
+  test('it renders placeholder string', async function (assert) {
+    assert.expect(2);
+
+    await render(hbs`
+      <DenaliMultiSelect @options={{array "Item 1"}} @onChange={{this.onChange}} as |option|>
+        {{option}}
+      </DenaliMultiSelect>
+    `);
+
+    assert.dom('ul.selections.has-arrow').exists('DenaliMultiSelect renders an wrapper ul');
+
+    assert.deepEqual(
+      findAll('.select').map((selection) => selection.textContent),
+      ['Select Items'],
+      'DenaliMultiSelect renders placeholder element'
+    );
```

Comment:
Can we test with a provided `placeholder` arg?

Reply:
added below